### PR TITLE
Create metal-iplay-ro.yml

### PR DESCRIPTION
**Pull request by xfouloux**
#2587 solved

iplay.ro is a metal tracker for music.

Tested on my preprod environment, 

image

### CODE REVIEW

**Comment by xfouloux** [comment]
well, download is still not working and I don't know why :/

File: src/Jackett.Common/Definitions/metal-iplay-ro.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      leechers:
+        selector: td:nth-child(7)
+        filters:
+          - name: regexp
+            args: '([\d]+) leechers' # get the first number from the result
+      grabs: # number of downloads
+        selector: td:nth-child(6)
+        filters:
+          - name: regexp
+            args: '([\d]+) times' # get the first number from the result
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/metal-iplay-ro.yml]: might as well add a hard-coded category in the results if you are not going to process a selector, and a few other default fields
```yaml
      category:
        text: "Music"
      downloadvolumefactor:
        text: "0"
      uploadvolumefactor:
        text: "1"
```
as for the download, looks like they sabotaged the html with an extra quote at the end, to make the href invalid
`<a href="download2.php?id=104687"">`
so you are going to have to either fix it with a filter or build a custom one by fishing the id out of it

File: src/Jackett.Common/Definitions/metal-iplay-ro.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        selector: td:nth-child(8) > span
+        attribute: title
+      size:
+        selector: td:nth-child(6)
+      seeders:
+        selector: td:nth-child(7)
+        filters:
+          - name: regexp
+            args: '([\d]+) seeders' # get the first number from the result
+      leechers:
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/metal-iplay-ro.yml]: i think your leechers is identical to you seeders ;-)

File: src/Jackett.Common/Definitions/metal-iplay-ro.yml (hunk 1 above)
Comment (garfield69) [reply src/Jackett.Common/Definitions/metal-iplay-ro.yml]: oops, sorry, nothing wrong with your leechers, just bad eye sight on my part ;-)

**Comment by garfield69** [comment]
I'm just going to floating ideas here, with no specific explanation for what is going on.
have you checked directly via a browser if the site requires you to click on a thank-you button before downloading,  or if it checks that you have sufficient download credit? odd behaviour for a semi-private site, but that is all i can think of that might interfere with the download.
alternatively, is there a way you can build the final href so as to bypass the download2.php and go straight to the download.php ?

**Comment by xfouloux** [comment]
well, there is no download.php or else, I did a change to force the login again just before download, it is way better, but there is still something not ok for Jackett, you can see the torrent content being downloaded and read, but still it give an error I don't get...

Download block added : 

`  download:
    method: post # use HTTP POST instead of GET to download the torrent file
    before: # HTTP request which needs to be done before downloading the file
      path: "login.php" # request target
      method: "post" # send via HTTP POST
      inputs: # list of HTTP arguments which will be included
        username: "{{ .Config.username }}"
        password: "{{ .Config.password }}"
    selector: "http://metal.iplay.ro/announce.php(.+?)"`

`10-11 18:51:12 Info Executing action method Jackett.Server.Controllers.DownloadController.Download (JackettConsole) with arguments (metaliplayro, Q2ZESjhCU0V3NUM2VHNCTXNreFBiVXNBRjB0Z29sblFqWnVlOVI5NFRUUDVpc2F5eTd4N2lXRC1CQ2kwZXBmbkRrWGVva2pHVXRkOUdlSkotNk5RcE05SmhFeHRDakxPZGpXTWFwempiMlRJeks5Nzhvdmp0OVJ1VnJBQ3lyZ3JpbDN6YU1mSk1KS2NTT0FfbTNXd0Y1M214Ymx4RXFJc1hFVUtGMHFKSDVkVTJ5Zlg, wauk7k4zlsziij2ot3i81dvlcq7jexfn, Bedemon-Child of Darkness_ From the Original Master Tapes-(Reissue)-[2...) - Validation state: Valid
10-11 18:51:12 Debug WebClient(HttpWebClient).GetBytes(Url:http://metal.iplay.ro/login.php)
10-11 18:51:13 Debug WebClient(HttpWebClient): Returning OK => 0 bytes
10-11 18:51:13 Debug CardigannIndexer (metaliplayro): handleRequest() remote server returned OK
10-11 18:51:13 Debug WebClient(HttpWebClient).GetString(Url:http://metal.iplay.ro/download2.php?id=104677)
10-11 18:51:14 Debug WebClient(HttpWebClient): Returning Redirect => http://metal.iplay.ro/download.php/104677/Bedemon-Child.of.Darkness.From.the.Original.Master.Tapes-(2005)-RMT.torrent
10-11 18:51:14 Debug WebClient(HttpWebClient).GetString(Url:http://metal.iplay.ro/download.php/104677/Bedemon-Child.of.Darkness.From.the.Original.Master.Tapes-(2005)-RMT.torrent)
10-11 18:51:17 Debug WebClient(HttpWebClient): Returning OK => d8:announce75:http://metal.iplay.ro/announce.php?passkey=XXXXXXXXXXXXXXXXXXXXXXXXXXX:created by13:uTorrent/182013:creation datei1539081831e8:encoding5:UTF-84:infod5:filesld6:lengthi1246e4:pathl11:CDimage.cueeed6:lengthi320341985e4:pathl10:CDimage.wveed6:lengthi88e4:pathl13:checksums.sfveed6:lengthi48422e4:pathl9:cover.jpgeed6:lengthi6870e4:pathl7:EAC.logeed6:lengthi1188e4:pathl8:info.txteee4:name66:Bedemon - Child of Darkness. From the Original Master Tapes (2005)12:piece lengthi524288e6:pieces12240:A?h?X?e?o?W??Pi?m????q1??:?+??
?UY?=z????'3?kD?{Y???U????w[y?o$?J??PWi??j??%?SXe????v???F?d??OC??M:a?|    q?? ???$xr??z???
b/??&X?Le7J?Y\
?CX(?K\v5???
??.os????X????d.0C?Z??an_{??{_D??~H???MM/??q??<6%?K???7-$?@81?mOMc?^3???]o?jj ?u??&=X?7g?^j6i?Ig??&??Tw)?H?:?PD???K0??J??^?????0???Og???3?w???j??M_??y???I?Z?L?&%c!    ?o??j???E??*?W??M?}C?p???e?????L4?g??L??????d?`??mLq???????]n????+????ß?,?tg?[:??       e+U??h?? ?Z???t?V?U?iz        ?79??I:w2??N0?=??7?{n??????g??/J????(?4?}??K???|}?so?S????)O:b5"???2??0?,q1'?m{?f?7x??>?p}????\?uw{nJ?????n@)?`??(??$!?[2e?3T?&T??P??$B????Z??pS?=???Z;?L???I??G??.B?#???B??,??W0zµ??z???:???V?V?? ???>??Z4??O?K??#???<?0??g??<????wEA5??s?F???C??ZR
?I?D?c?z?O?'A?!?P ?D?kz?6??A?? ob?,>??#]??ov&???|v??$??7?SH?)9?/+Nk??[??.#w?)?}?@?t?f?????H??"?WP????Y]??????m?>GÚJ???C)?P8?c??sb?+???Owv&mkG?-?R??X?\<k]??T?*M?????2z?t?:4T?5????]g?|?%_{?5y??}??\?woGKJgK?p?B??????????O5%?J8O?:g =7(??]??C???iZ -?:%=f?J?)B1s?`^?-???????if?i?$?Q???w?M?pP/???~b?T?I?3?>O?EDT???#-D?+??[??J91??;??%d:?u???6??+?e?=bk??R???2??      xZ}OcA??.?F@???f!d?P???Q?/?Je?}:?h???q?;???T??YdmV?+??1?!?=.N\$?t?$?6???T??rC=dxU^??+?????G?`??Nn%?N?
kXN??oUN?6n@???!????+?m6???"aM????rP?J?I;?A?s?r??y3?C?.????G?B??M?WpU?r[     ????&?'??/"%?U???nqG???C????x?k_??Q?^?t¨????y,?[?R????     ??C??E?k??'*???/???7?#??D?L?w>iq?^?XTB[>?n.?s?6?TS?+?P?ks3?Y???n\???[??,?
?mk?[-?4>?>=?C??d???A??A.???+??/?ce??:??z??`Y??2;2xC?L???f?p??Y?@??zN????T?F??jM????hg? ?4?n??~Z???D???"??R??8??,E[??q-?C?????uv~??q?
??|^m8?yj?C?%C?b@??????l"??N????%?w???.?lEj??    ???T???3       ?xei?pnt??J?w|????}]
????(?S?J8o?j??mNVt?2{????:7?g??Tm~a.n?????[c?5w???>???~?64????T?N?r???W???T??w????4??/q????o?R??}&???Y5Kd>(l!j?????!?m?|???????Yy??zV????? ??:??BnQV??7??T??v/??}qc???r?u      j?<?RC???3?(????~h??e??F2z???W?W?r ?f1?_??Q????\????ooWVSxG??Y??r?fDK?]???g~r???x??]p4i;s??v??rIgb?[m???T?F[??6??????L?x?????j?????y\N???ns??]?<\??????C????]??]@0???|V?j\???U7?'??n?au??I?BM???@?\?F??D??????iL?ae??N?????*???`E??9DL?(???0n????L       ?|?e?.?.k????Os/??Ys??w7B9?,?E?-
????q????????*???:?ly????W?{?,f???x??W?4?Q       Pr?4???M?l@????,}>T??W~????h?1?h?iu_???<?\JF?e????Oh??Mb??3Cwk?:d?d???????4|????*?\?  ?#C??d??e?W?R?B@???f?1 4???{???t??+???~p^a??M5?S??3?C??^N???8l??,?f???>
?XV?P?:?yr?&?O?I??V~Hs?v?A?q??
?       ??w??QP??Kc??Kx?D??Z?J?/,S????E?l?????GD<t-7?=u?gv??u?/???-?{;?d??????r?u?Z?CK?s?????jh4?%??-pNK?T?j?b????? ? t??????&A??o??i6?/?KSP9?a]?N2qU?E{M?(?Y?0
=?L+l??5??Z?C??#5WZ?wa.??v?s\??>??<I5B/X?(?,???xU(????=i?\?o?m???\???=?????B????YS?r??#????:????E????[?}Xh???n??6xn??z?y??!_??g???????v(??o??MzV?Y???H???@?)??z _??$(??V??@H|XX'??}\
dn??,?c?N?gÐ??????NH(?R?|p?rY7R??=????????'?W6c??M?4|?FPl?K??????\b???_?w:v???OPl?:=ke?O??_?o????h[?? ?A!Q?q?+???T?z{[?:;~?4?h??1?x??B?p?L?@Pc??St???    ?1R?????? ?J???RLhA1?+?%?????l??h&?V????i^????'??s?1?J?Y???Rr=???"O??z?????????cH??U????|?'?n`??(?)??Z?In??m?g?R???b?(iT??\mX?lRD??a?b???c??g?;?u?a3??y???+??-?H????P?*7BANF0~????"??%???M??5?E?gm???FN???T ??`a??"?g?:zs-???!?í???z|????B'?<ZK?6??Z??T?6?J+???KM?A??i(?Q?4"+?Y,
?J?R??$9Eb,???+Fe.?vBIG?Z??[????Sh??O???????q/U^????!?Z?<??5??MF?~ ????????+?q; t??oB@?D??z?B?U.??b61?@99:~"?N+???-A??_??????B#N\I???Ia8h35?y??t]|L?k #):S???9:i?Y?MAQ#]G:?DjM2r?ea??g&fn!?6L?A? YO??!?? ???q??X???@?Sh?:X????RH?;??W?ot.?B?w?*
4.????f?T?&?a8#?S???????????P|(}??
`s??Wh?????g?N?2nJc?a?+???¤q?H??????3??~?]#????4gX5}*??%6HV?|????HDH:?'??P????P?(v?'?S ?u?m\?5?3??cA<?(^q?6?'/3?[?<?????p??-?`?\???l?l??Mz5}Y?"?s?o?¦3??^-??3'?$"7?a?kM?Rz?H,#"??????1B$?%]???uP?Q?4TGmNm??"?????
I4??;??k?
|%??nD)D{ Z???UA3???bF?f
?,?m?A4H8?0??\???????6????qm]dy???Pd??}#??9?sP???s??|???u2?j???\MMu,Qr?3hN?{????U?]#&?0?L??????x?8?Y???5?,?
U?,?Tj?>??7K??\L???k?bz?<?d??k(??K??i6?????????1]?
7???w?a??"Y??M???M???RN&@C???u?wB?*t??)??2PG?P?v?*ZnW??=b??h???Z?? ????5*5?=.???`w??_?=Zk????lO*k????e???c)??>??H{nx???H?A?H??P?Q???-??J???3?(S??L??l7??>?_?O???@?????(??mw???Q????s???~9??J??U\??(??C???1?????F!???T?N??YK+v<?q$?t}???U????????h?L???Q??????}??uI??d-?
4?v'.q?m? ????'??6$PTK?&??)??f?_?t?????.JF^?????&???|4
D???GZN?*?}??5{???U?aj?z?<?]HS?????V}?+????Io??%???jC???fn, j[????N?_N????,Q?}?????ml3??L?<uU?&2??&???N:?9?}?OI??z?F??2#??`q6??(}q??9?S?i#?D????:??D?P
?]?%V?4?-f??h?k?3G ???I??{??]_?Y'????M????re&W8?;`l??;?+?n? ??{???p??? ??????v|???^9???c0T?EP|Pd|?|?m?|Js???yiPR????6b,/???T6??a?X?W?D?6?8???_?jiS????g?I??Ml?~q?3??wo?v?.?2?F?q?o??x?aU??J|'q«M??
2C?2%¸?~?"???a??t?????Ö:,&?Vk?<?|?        Q?A?T??%i9?d4D?Zuy>??X??E?????2_|^*  ????vs?v??T?h?&\??_)d??¾ ?,?h1»??e???=]??9???IC?v?jA*??3???R???8d?"o?-??P_?^{?B??G?]w?nl5???????&?}26??gooPu`T?aW?e?U?? ?'?0=?M"?????m??Qx?)Z?J+?A?O-????apiqkA>????(U?x?T????'?_       ?a?????of????p?K??g??o?F?q;%?l????+?NW?)_5$?dE???z??\??YZ6?Q?G?q?@y?t?_    ?m}??>?+#?TZ!?b?Bd?K?|?`t)?|3,AO???=?????>?:?E?8???*???ag+#???gQ???gWf???$?ain?F???????7?bt???9*?d??{x?l^.@]??/?F?% ;
v??U?e?ZQd>kV[???D?9%??eR?x#x.?w?? ??E???:??M????????(%s
2?&k??^?5p?Y ???K?"tX{???oGf??????ã2??????`#S?2?8uS?t???@9?AE??X??G?2L????F?Yc??g_&`H??"kPXSp3u??U???&??FWo??2u?$E?xf?7nF?????BG]l<?`??d??i:M?P?y?t????Z??F??k?wa?H?|(6U?d9?g?}Kd??A?????F{".?X}\'?,???eV#??1:?&a????s????|?;A????s?*?????l0nw*??????CNE?|???k???%?%?c_?? u?9??b>?wn?p  2z?E? >/l(??y??
5? 6G????z?}??V{n??b?Y?H?H ???S?5?x?[??kj??2vDf7???#O+?t5vO?e?????3?08fQxwE>?,7Ug?OeF[????YA?3)??+$?B??&V9?????W?X?_!?_o-?C_??j??Z1(?@nP?=dW?b?QcL?????"D?7?e?(-)???r={8?[??pD'(?]I??q???JVo??~??W?h?`R????X??X{c?Q?90p?n???m=?_+???^??v\T0??n??w??? ??G|4??4s/?m?abhN ???nsE~O?@?????i`8o?'????{CNOr?s8?;BH??%?m?A?6!??uq?r????
???o?G?F?8??T???4?A??
?5=????8h???s???    ????z?????B???????|z?uw?C?8??44/?F¸??[~??x??6???? ?-?????v}?_??r?l?fF???S??;??Ua???w\/??????!?????[Ur@X???q?2"'1?{??#K}???????+6?+m??Mi?!?????#W?}?^w$?^??t?}?q`???
wT?^??f????4D??V??7a-???8?mt+?L?q8(?y????B?r,_??9??(>?Ka??ibK|}???z???q???j?$i&Ym?????,?w????y?7?"?UF?v?w????6X??8O??u?J
&??~?? |????N?c]!?g??
?U?I????Cee7üLi?u??'?4?>5?K???q?a3?XM?1?+??1?'??3?|? y?r????7??3OR?w??lRgH%l ?B?Gv??lh?b???V???x??>        *???=?N??b?v"?4lWN?NHoA?
??ht????'A?AL??=??'F?'?a??r??8#?D$???kn*3:?f??4=w9??I?+?ksA??p??6??E"I+B????A?F{?!t-??I?1??>
K~?????t?$      ?????o .Z????????`{???&RT??|]5?:?#c?[???;q???A]?+?/?e??????^??N?:~??'?=+??r?x???<?>?I?o*??Jb?L??Wbd?r?b#Fj?n)p??#?d???c????r?lJRYi?Q??[1!??????*?????=?&{?C????H??b?Iour9????ac??d( ??????u?y?????]?4??c\??;???9?#????.?????(?3?[6?????I{?zR?????/??l????&ui9Q?,R?-????I???!
sp???????`De?C?/?F?G/Z???/H?????z????U?0?????"??l      fg??\%|?"?k??G ??c?$??????????-??-C0???i?X(???????[???X???;/??{????q?~??}? ???fe?b??"?O???+S?A?gR???z?.??S\u??1?c???%??U? {?c`?;Á????.?W??? ?*+?P)V??L??????7=???????g?????/?????1>5???N?f"???_?K??gsh$V?b?????o??LAY?f?-D??????!??] >(??T????D?w7ak?|T:?
?O&?Y?h^_???KD%???0_?%??
Y???>>|ki?I?#?? ?+;????&7??Q?<??rn?$?d??I??x???R??L???[;?:??<?f?`"s??!?i?|???8a?9?"???????Y?I???=''6??z^?'
K?r?w7eZq? H?;v?h??F.0??M'?y????GBy?_a?Pm?3`4???y       ?[??????p??>B???V??????9a98? ???w$s8?#??;?w}???R?A?L??      ???b{CQ<WOFp?(
P??????R?tm??G??>????l?b?S??a?K?[?=@???????a^92|FN??
]x?gW"Wm?4t????T?Yyp???H7???n?????1?>????\??J??????/??;?V?A?kQy??m ??}???"?g?i?\???>??tÔ?{????wt?+?????#???f?2???? B?????f?A??????? ?8?^B??1??:?????r???????y?????J?-n??'?R@G$???d???]?o?Á(?-W?W?A?tC"&?D??]? b]??q?*ko<????\#?\K?-p??75[?L?iC8??F?#?i? ?%?J?W?d?N??F?Y???~???j????? dY??? ????+p?h??4????>e?
G??q<????$?   ??????????Ga??X?U?
n? EK?t???&(PI1
????Ù?h??~???PP????7fl?R?????x?=S?J?F?_??V?^4:F:??yO4/??G5?C??"?K??@:???I?I?_Nm????e???@;??j"`Zip??V/O?2q?????}???Q ?lrc????}!Jl??,D9kc ??DK?$ \???j??$
??      `CKK?{g?????VU?}??C?Y?na?7?D???!W?e^7:privatei1e6:source40:Romanian Metal Torrents 12 years editionee ???"?<_???"?X??jÊ??PT?c/b???u???   ????/???0?@??bIr??X??kI4L?? ??D?? W??9??
10-11 18:51:17 Error Error downloading metaliplayro http://metal.iplay.ro/download2.php?id=104677 AngleSharp.Dom.DomException: The string did not match the expected pattern.
   à AngleSharp.Extensions.QueryExtensions.Validate(ISelector selector)
   à AngleSharp.Extensions.QueryExtensions.QuerySelector(INodeList elements, String selectors)
   à AngleSharp.Dom.Document.QuerySelector(String selectors)
   à Jackett.Common.Indexers.CardigannIndexer.<Download>d__32.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à Jackett.Server.Controllers.DownloadController.<Download>d__5.MoveNext() dans C:\projects\jackett\src\Jackett.Server\Controllers\DownloadController.cs:ligne 53
info: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[2]
      Executed action method Jackett.Server.Controllers.DownloadController.Download (JackettConsole), returned result Microsoft.AspNetCore.Mvc.NotFoundResult in 5374.6435ms.
10-11 18:51:17 Info Executed action method Jackett.Server.Controllers.DownloadController.Download (JackettConsole), returned result Microsoft.AspNetCore.Mvc.NotFoundResult in 5374.6435ms.
info: Microsoft.AspNetCore.Mvc.StatusCodeResult[1]
      Executing HttpStatusCodeResult, setting HTTP status code 404
10-11 18:51:17 Info Executing HttpStatusCodeResult, setting HTTP status code 404
info: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[2]
      Executed action Jackett.Server.Controllers.DownloadController.Download (JackettConsole) in 5466.289ms
10-11 18:51:17 Info Executed action Jackett.Server.Controllers.DownloadController.Download (JackettConsole) in 5466.289ms
info: Microsoft.AspNetCore.Hosting.Internal.WebHost[2]
      Request finished in 5601.8147ms 404
10-11 18:51:18 Info Request finished in 5601.8147ms 404
`

**Comment by garfield69** [comment]
I just tried accessing http://metal.iplay.ro/ but I get redirected to https://metal.stream.bike/
which is a private site so I cannot do any research unfortunately. 
Its possible they have a Romanian IP-only filter.
but if you cannot figure out what is going on then maybe @kaso17 can take a look, he is far more experienced at this than I am.

**Comment by xfouloux** [comment]
Humm indeed the url have changed, but only in the last 24hours, yesterday it was still metal.iplay.ro
They do not have an IP filter, it is a private tracker.

I have 3 invites so I can share one if you want.

**Comment by xfouloux** [comment]
ok last commit works now !

**Comment by xfouloux** [comment]
Hey, is it possible to use a filter for the case in the downloadvolumefactor ? like this ?

```
downloadvolumefactor: 
        selector: rmttagscontainer
        filters:
          - name: regexp
            args: 'img src=(.+?) >'
          - name: strdump
        case:
          'freedownload.gif': "0"
          'half.gif': "0.5"
          "*": "1"
```

here is the HTML row, you can see there is a pic 'pic/freedownload.gif' when the torrent is freeleech, and 'pic/half.gif' if it is only half counted.

```
<tr class="torrents_table_row">
    <td class="torrents_table_row_categ" align="center" style="padding: 0px;background:black;">
        <a href="browse.php?cat=39&amp;&amp;incldead=0">
            <span style="background-image:url('pic//category/new/pic_death.gif')" class="imgover_2011" title=" Death Metal, Technical Death, Thrash/Death"/>
        </a>
    </td>
    <td align="left" style="line-height:12px;" onmouseover="this.className='rand_modif'" onmouseout="this.className='rand'" class="rand">
        <a href="details.php?id=104690">
            <b>
                <span class="regular_torrent">Imprecation (USA)-Satanae Tenebris Infinita-[2013/FLAC/lossless/cd] - RMT</span>
            </b>
        </a>
        <span class="rmttagscontainer">
            <a class="rmttags" href="browse.php?search=&amp;tlt=50&amp;incldead=5">
                <img align="right" src="pic/req.gif" title="Requested torrent" border="0/">
                </a>&nbsp;<a class="rmttags" href="browse.php?search=&amp;tlt=50&amp;incldead=6">
                    <img align="right" src="pic/freedownload.gif" title="Download not counted" border="0/">
                    </a>&nbsp;&nbsp;<a href="browse.php?search=&amp;tlt=50&amp;incldead=4">
                        <img align="right" src="pic/lossless_nonscene.png" width="12" title="audio only - Lossless NONSCENE" border="0/">
                        </a>
                    </span>
                    <br>
                        <span class="bandpropaganda" style="color:rgb(120,120,120);font-size:10px;">
                            <a title="Check out other Imprecation (USA) torrents." target="_blank" href="browse.php?search=Imprecation+&amp;tlt=50&amp;incldead=2">
                                <font size="1" color="#8faec0">Imprecation (USA)</font>
                            </a>
                            <img class="rmtdot" src="pic/info/dot.png"> Black/Death Metal | USA: Missouri City, Texas | Formed In 1991...</span>
                            <span class="rmtripformat" style="color:rgb(84,118,69);display:inline;">
                                <img class="rmtdot" src="pic/info/dot.png"> tracks+cue+log</span>&nbsp;</td>
                            <style>
        #ttablelinks img {margin-left:4px;margin-rigth:4px;}
        .rmtdot {width:6px;margin-left:3px;margin-top:-1px;margin-bottom:1px;}
        .rmttagscontainer {float:right;margin-top:-4px;}
        .rmtripformat {display:table;float:left;font-size:10px;font-style:normal !important;margin-top:4px;}
        .rmttags {}
        .torrents_table_row {height:40px;}
        .bandpropaganda {display:table;float:left;margin-top:4px;}
                            </style>
                            <td align="center" border="0">
                                <div id="ttablelinks" style="display:table;width:64px;;text-align:center;">
                                    <a href="viewnfo.php?id=104690&amp;view=fonthack">
                                        <img src="pic/info/nfo.png" border="0" title="View NFO">
                                        </a>
                                        <a href="download2.php?id=104690" "="">
                                            <img src="pic/info/download.png" border="0">
                                            </a>
                                            <a href="bookmark.php?torrent=104690">
                                                <img border="0" src="pic/info/bookmark.png" alt="Bookmark" title="Bookmark">
                                                </a>
                                            </div>
                                        </td>
                                        <td align="right">0</td>
                                        <td align="center">284.17 MB
                                            <br>8 times</td>
                                            <td align="center" style="font-size:10px;">
                                                <b>
                                                    <a href="details.php?id=104690&amp;hit=1&amp;toseeders=1">
                                                        <font color="">5 seeders<br>
                                                            </font>
                                                        </a>
                                                    </b>
0 leechers</td>
                                                <td align="center">
                                                    <a href="#">
                                                        <i>[Anonym]</i>
                                                    </a>
                                                    <br>
                                                        <span title="Date of upload: 2018-10-10 22:29:48" style="cursor:help;">1 day</span>
                                                    </td>
                                                </tr>
```

**Comment by garfield69** [comment]
Thank you for  this and rockbox. 
Cheers.

**Pull request merged**